cas-server-support-osf/src/main/java/io/cos/cas/web/flow/OpenScienceFrameworkLoginHandler.java

```diff
@@ -83,40 +90,32 @@ public void setHandleErrorName(final String handleErrorName) {
             this.handleErrorName =handleErrorName;
         }
 
-        public boolean getInstitutionLogin() {
+        // Must be public to be accessible in the JSP page
+        public boolean isInstitutionLogin() {
             return institutionLogin;
         }
 
-        public void setInstitutionLogin(final Boolean institutionLogin) {
```

Comment:
An interesting non-bug. `Boolean` is a reference type. This doesn't break anything tough. I should be the primitive type `boolean`.